finagle-redis/src/main/scala/com/twitter/finagle/redis/SentinelClient.scala

```diff
@@ -0,0 +1,85 @@
+package com.twitter.finagle.redis
+
+import com.twitter.finagle.builder.ClientBuilder
+import com.twitter.finagle.redis.protocol._
+import com.twitter.finagle.{ Service, ServiceFactory }
+import com.twitter.util.Future
+import org.jboss.netty.buffer.{ ChannelBuffers, ChannelBuffer }
+
+object SentinelClient {
+
+  /**
+   * Construct a sentinel client from a single host.
+   * @param host a String of host:port combination.
+   */
+  def apply(host: String): SentinelClient = SentinelClient(
+    ClientBuilder()
```

Comment:
Daemonize is already true by default in the finagle 6 API.

there's  no need to configure DefaultPool in the finagle 6 API because it uses the pipelining pool, so there will only ever be one outstanding connection anyway.

LGTM otherwise!